feature_tests/js/api/diplomat-runtime.js

```diff
@@ -0,0 +1,93 @@
+function readString(wasm, ptr, len) {
```

Comment:
Issue: I think you didn't delete the .mjs version of this file as it looks like you intended to do